app/services/current-user.js

```diff
@@ -9,13 +8,17 @@ export default class CurrentUserService extends Service {
   @service
   store;
 
-  load() {
+  async load() {
     if (this.session.isAuthenticated) {
-      return this.store.queryRecord('user', { me: true }).then((user) => {
+      // use existing record if its already loaded
+      let user = this.store.peekRecord('user', this.session.data.authenticated.user_id || '');
```

Comment:
user_id may still be blank here if the user is logged in with an older version of the code that didn't include the user_id. see: 
https://github.com/datafruits/streampusher-api/pull/110